www/js/diary/infinite_scroll_list.js

```diff
@@ -45,11 +45,13 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
   $scope.data = {};
   // reset all filters
   $scope.filterInputs = [
-    InfScrollFilters.UNLABELED
+    InfScrollFilters.TO_LABEL
   ];
   $scope.filterInputs.forEach((f) => {
     f.state = false;
   });
+  $scope.filterInputs[0].state = true;
+  $scope.allTrips = false;
```

Comment:
Line 53 sets the first filter tab to be active by default; line 48 specifies that that's `To Label`. Line 54 declares a variable keeping track of whether the `All` tab is active so we can highlight it or not just like all the others. I need a separate variable because `All` is not one of the `filterInputs`. I could use a function that checks whether all of the `state`s in `filterInputs` are `false`, but you advised that I should use properties instead of functions for CSS classes. I can add documentation for this.

Reply:
yes, please. I figured out most of that from the code, but making life easier for others helps with maintainability. Even just a note saying that this is related to highlighting the all tab (turning it green) would be helpful because it will help others know what to expect in the code and find the parts that match their expectations.

But again, you can defer this to a later commit.